server/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const knex = require('./knex');
+const port = 3001;
+
+app.set('port', port);
+
+app.use(cors());
+app.use(express.json());
+
+
+app.listen(port, () => {
+  console.log(`Server has started on port ${port} 💸 🔦 💫`);
+});
+
+// ROUTES
+
+app.get('/api/v1/entries', async (req, res) => {
+  try {
+    const entries = await knex.select().from("entries");
+    res.status(200).json(entries);
+  } catch (error) {
+    console.error(error.message);
+  }
```

Comment:
Overall just super proud of us.

Reply:
++ Joe! So proud of us! Thanks Chrissy for driving.

Reply:
++ great work team! don't have a lot of comments, just trying to grasp everything at the moment